features/step_definitions/common_steps.rb

```diff
@@ -235,7 +235,9 @@
 
 When /^I enter #{MAYBE_VAR} in the '(.*)' field and click the selected autocomplete option?$/ do |value, field_name|
   field_element = page.find_field field_name
-  field_element.set value
+  field_element.set ''  # clear the field before typing
```

Comment:
Is there a possibility that these fields are not clear to start with?

Reply:
Yes, in the edit company details journey where it's an admin changing a suppliers registered address the field is populated with the existing data.

So otherwise the autocomplete will be sad because 'FranceIreland' is not a country 😂

Reply:
_I'm_ now sad FranceIreland is not a country...

Thanks for clarifying